src/RecoveryServices/RecoveryServices.SiteRecovery/ReplicationProtectedItem/NewAzureRmRecoveryServicesAsrReplicationProtectedItem.cs

```diff
@@ -289,6 +292,9 @@ public class NewAzureRmRecoveryServicesAsrReplicationProtectedItem : SiteRecover
         /// </summary>
         [Parameter(ParameterSetName = ASRParameterSets.AzureToAzure, HelpMessage = "Specify the proximity placement group Id to used by the failover Vm in target recovery region.")]
         [Parameter(ParameterSetName = ASRParameterSets.AzureToAzureWithoutDiskDetails, HelpMessage = "Specify the proximity placement group Id to used by the failover Vm in target recovery region.")]
+        [Parameter(ParameterSetName = VMwareToAzureWithDiskType)]
+        [Parameter(ParameterSetName = VMwareToAzureParameterSet)]
+        [Parameter(ParameterSetName = ASRParameterSets.HyperVSiteToAzure)]
```

Comment:
Similar to RecoveryAvailabilityZone, could you add HelpMessage?

Reply:
> Similar to RecoveryAvailabilityZone, could you add HelpMessage?

Done